ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -5,7 +5,7 @@
 
 declaration template components/ssh/schema;
 
-include 'quattor/types/component';
+include 'quattor/schema';
```

Comment:
There's nothing extra needed, but we don't have quattor/types/component, presumably because our template tree is based on an old template-library-core and it changed at some point.

I'm not against changing things on our end, but the reason I changed it is because include quattor/schema seems to be the much more common preamble across configuration-modules-core.

$ grep -r 'include.*quattor/schema' */src/main/pan | wc -l
63
$ grep -r 'include.*quattor/types/component' */src/main/pan | wc -l
9